domain/src/main/java/edu/nextstep/camp/calculator/domain/Expression.kt

```diff
@@ -0,0 +1,77 @@
+package edu.nextstep.camp.calculator.domain
+
+
+class Expression {
+    private var expression = ""
+
+    fun add(operand: Int): String {
+        if (expression.lastOrNull().toString().isOperator()) {
+            expression += " "
+        }
+        expression += "$operand"
+        return expression
+    }
+
+    fun add(operator: String): String {
```

Comment:
```suggestion
    fun add(operator: Operator): String {
```
Operator를 정의하셨으니, Operator를 활용해보면 어떨까요?